.eslintrc.json

```diff
@@ -4,15 +4,27 @@
     "next",
     "next/core-web-vitals",
     "prettier",
-    "plugin:storybook/recommended"
+    "plugin:storybook/recommended",
+    "plugin:@typescript-eslint/recommended"
   ],
+  "parser": "@typescript-eslint/parser",
   "overrides": [
+    {
+      "files": ["**/*.mjs"],
+      "rules": {
+        "@typescript-eslint/no-empty-function": 1
+      }
+    },
     {
       "files": ["**/*.ts?(x)"],
       "plugins": ["@typescript-eslint", "no-relative-import-paths"],
       "rules": {
         "@typescript-eslint/consistent-type-imports": "error",
```

Comment:
I assume this rule and the one from the next line are already part of the recommended rules?

Reply:
This rule is now removed.

Reply:
Yes it is part of recommended rules.